frontend/src/metabase/components/SegmentedControl.styled.jsx

```diff
@@ -19,8 +19,7 @@ const COLORS = {
   "fill-background": {
     background: ({ isSelected, selectedColor }) =>
       isSelected ? color(selectedColor) : "transparent",
-    border: ({ isSelected, selectedColor }) =>
-      isSelected ? color(selectedColor) : getDefaultBorderColor(),
+    border: ({ selectedColor }) => color(selectedColor),
```

Comment:
`fill-background` is a default `variant`, so it seems to change most of the `SegmentedControl` looks
Is it an expected change or do we want to do something to continue using `color("border")` most of the time?

Reply:
The last time I had to change the style of the fill-background variant of the segmented control, @mazameli encouraged me to make it app wide so that things would stay consistent. I'll see if I can find the slack thread, but making it the app wide was deliberate

Reply:
https://metaboat.slack.com/archives/C03SCHEU16W/p1664401889241849?thread_ts=1664397118.624389&cid=C03SCHEU16W

Reply:
Cool, thank you!